doctr/utils/geometry.py

```diff
@@ -27,7 +27,7 @@ def rbbox_to_polygon(rbbox: RotatedBbox) -> Polygon4P:
 
 def fit_rbbox(pts: np.ndarray) -> RotatedBbox:
     ((x, y), (w, h), alpha) = cv2.minAreaRect(pts)
-    return x, y, h, w, 90 - alpha
+    return x, y, h, w, 90 + alpha
```

Comment:
Mmmh, is this a good idea?

Because the following snippet:
```python
import numpy as np
import matplotlib.pyplot as plt
from doctr.utils.geometry import rbbox_to_polygon, fit_rbbox

angles = np.arange(-90, 91)
rboxes = [fit_rbbox(rbbox_to_polygon((100, 100, 40, 10, angle)))for angle in angles]
estimated_angles = [rbox[-1] for rbox in rboxes]
estimated_width = [rbox[2] for rbox in rboxes]

plt.plot(angles, estimated_angles, label="angles")
plt.plot(angles, estimated_width, label="width")
plt.xlabel("Anti-clockwise rotation")
plt.legend(loc="upper left")
plt.show()

```

now yields
![troubles](https://user-images.githubusercontent.com/76527547/146214652-6d2dcb17-0869-40a1-a265-173dff56ee59.png)